sdk/spring-sdk/src/main/java/kalix/springsdk/annotations/Subscribe.java

```diff
@@ -26,4 +27,11 @@
   @interface ValueEntity {
     Class<? extends kalix.javasdk.valueentity.ValueEntity<?>> value();
   }
+
+  @Target(ElementType.METHOD)
```

Comment:
Note the work in progress on allowing this at service/class level as well, can probably be a follow up but just so you know. See PR over here: https://github.com/lightbend/kalix-proxy/pull/1622/files#diff-c1390e2e16a5479ebfa65460b72ddb4985714f0c0a3a9727be20e50b1af22502R46

Reply:
I suggested to remove the `ElementType.TYPE`, but didn't think about the case you mentioned. 

That will be indeed much convenient to annotated only on the class.